hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7953,6 +7954,7 @@ private WriteEntry doWALAppend(WALEdit walEdit, Durability durability, List<UUID
     if (walEdit.isReplay()) {
       walKey.setOrigLogSeqNum(origLogSeqNum);
     }
+    walKey = this.coprocessorHost.preWALAppend(walKey);
```

Comment:
Can `this.coprocessorHost` be null here?

Reply:
Wrapped in a null-check.